node-mongodb-high-availability/azuredeploy.json

```diff
@@ -0,0 +1,561 @@
+{ 
+    "$schema": "https://schema.management.azure.com/schemas/2015-01-01/deploymentTemplate.json#",
+    "contentVersion": "1.0.0.0",
+    "parameters": {
+        "adminUsername": {
+          "type": "string",
+            "metadata": {
+                "description": "Administrator user name used when provisioning virtual machines (which also becomes a system user administrator in MongoDB)"
+            }
+        },
+        "adminPassword": {
+            "type": "securestring",
+            "metadata": {
+                "description": "Administrator password used when provisioning virtual machines (which is also a password for the system administrator in MongoDB)"
+            }
+        },
+        "storageAccountName": {
+            "type": "string",
+            "metadata": {
+                "description": "Unique namespace for the Storage Account where the Virtual Machine's disks will be placed (this name will be used as a prefix to create one or more storage accounts as per t-shirt size)"
+            }
+        },
+        "location": {
+          "type": "string",
+          "defaultValue": "West Europe",
+            "allowedValues": [
+                "West US",
+                "East US",
+                "East Asia",
+                "Southeast Asia",
+                "West Europe"
+            ],
+            "metadata": {
+                "description": "Location where resources will be provisioned"
+            }
+        },
+        "virtualNetworkName": {
```

Comment:
Ok, but what if the user wants to use an existing setting or has a naming convention? There is  a default value if not. What do you think?